azurerm/provider.go

```diff
@@ -233,7 +233,11 @@ func Provider() terraform.ResourceProvider {
 			"azurerm_container_group":                                    resourceArmContainerGroup(),
 			"azurerm_container_registry":                                 resourceArmContainerRegistry(),
 			"azurerm_container_service":                                  resourceArmContainerService(),
-			"azurerm_cosmosdb_account":                                   resourceArmCosmosDBAccount(),
+			"azurerm_cosmosdb_account":                                   resourceArmCosmosAccount(),
+			"azurerm_cosmos_cassandra_keyspace":                          resourceArmCosmosCassandraKeyspace(),
+			"azurerm_cosmos_mongo_database":                              resourceArmCosmosMongoDatabase(),
+			"azurerm_cosmos_sql_database":                                resourceArmCosmosSQLDatabase(),
+			"azurerm_cosmos_table":                                       resourceArmCosmosTable(),
```

Comment:
can we make all of these:

```suggestion
			"azurerm_cosmosdb_table":                                       resourceArmCosmosTable(),
```